lib/ex_ami/client.ex

```diff
@@ -380,6 +385,8 @@ defmodule ExAmi.Client do
     end
   end
 
+  defp validate_salutation("Asterisk Call Manager/7.0.1\r\n"), do: :ok
```

Comment:
Given the format of the 2.10 handler I suggest you follow that pattern. This should check for 7.0.n, so that if UCX changes the version we don't have to come back here again just to sort .2.

I also think that we should now reverse the order of the values. Since 7.0.n will become the standard release it might as well be handled first in our list, and the other issues tested in decreasing order.

Reply:
I agree with @Sinc63

Reply:
Removed them.

Reply:
Made the change.